public/app/features/datasources/partials/tls_auth_settings.html

```diff
@@ -17,7 +17,7 @@ <h6>TLS/SSL Auth Details</h6>
 
       <div class="gf-form" ng-if="current.secureJsonFields.tlsCACert">
         <input type="text" class="gf-form-input max-width-12" disabled="disabled" value="configured" />
-        <a class="btn btn-secondary gf-form-btn" href="#" ng-click="current.secureJsonFields.tlsCACert = false">reset</a>
```

Comment:
nit: I think this might be a bit worse from an a11y perspective? Can we use a button instead? @kaydelaney WDYT?

Reply:
You are right, I changed the anchor element to a button.